### ISSUE TITLE
button contains empty `aria-checked`

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#198228]
whether or not a button is checkable, it currently has `aria-checked`, which is an accessibility issue and violates accessibility standards.
